src/features/Staff/components/StaffLinks/StaffLinks.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { memo, useMemo, VFC } from 'react';
+import { HStack, Link } from '@chakra-ui/react';
+import { StaffLinks as Links } from '../../../../models/staffLinks';
+
+interface Props {
+
+  /** Object with staff links. */
+  readonly links: Links;
+}
+
+/** Staff links component. */
+const StaffLinksComponent: VFC<Props> = ({ links }) => {
+  const linksList = useMemo(
+    () => Object.entries(links).map(([key, path]) => path && <Link key={key} href={path}>{key}</Link>), [links],
+  );
```

Comment:
I think there is no profit in using `useMemo` since `links` is the only prop here and you are using `memo` for component already.